backend/src/main/java/com/bakdata/conquery/models/auth/entities/User.java

```diff
@@ -29,13 +29,18 @@
 public class User extends FilteredUser<UserId> implements Principal{
 	@Getter @Setter @MetaIdRefCollection
 	private Set<Role> roles = Collections.synchronizedSet( new HashSet<>());
+	
+	/**
+	 * The name of a user is unique in the system. It is used to generate the UserId object.
+	 * The name might be an email address.
+	 */
 	@Getter @Setter @NonNull @NotNull
-	private String email;
+	private String name;
```

Comment:
Kommentar, dass das nicht Vor+Nachname ist

Reply:
Speichern wir in einem string